locales/en-US/app.ftl

```diff
@@ -446,7 +446,8 @@ MenuButtons--metaInfo--logical-cpu =
        *[other] { $logicalCPUs } logical cores
     }
 
-MenuButtons--metaInfo--main-process-started = Main process started:
+MenuButtons--metaInfo--main-process-started = Start:
```

Comment:
Please wait for @canova to confirm if we want to make this change. If we want, changes to existing strings require a new string ID.

P.S. also, revert chages to the localized files, looks like your editor trimmed trailing whitespaces.